app/bundles/LeadBundle/Translations/en_US/flashes.ini

```
@@ -16,3 +16,4 @@ mautic.lead.list.notice.batch_deleted="%count% lists have been deleted!"
 mautic.lead.list.frequency.rules.msg="No "
 mautic.lead.batch.import.created="Import process was successfully created. You will be notified when finished."
 mautic.import.file.missing="File '%file%' cannot be found in the filesystem. Please try again."
+mautic.lead.export.being.prepared="File is getting processed. The file will be send to your email address '%user_email%' You can download the file from the link provided in the email."
```

```suggestion
mautic.lead.export.being.prepared="File is queued for processing. The file will be sent to your email address '%user_email%' You can download the file from the link provided in the email."
```

We should ensure we are setting expectations for the users and tell them it is queued for processing. Small grammar fixes also :)